games/common/components/StepperOptions.js

```diff
@@ -216,6 +235,19 @@ define([
 				docutil.updateAttrs(this.maxFrame, {'disabled': 'disabled'});
 			} else if(this.maxFrame !== docutil.document.activeElement) {
 				this.maxFrame.value = config.maxFrame;
+				if (this.skipFrame.value > this.maxFrame.value) {
+					this.skipFrame.value = this.maxFrame.value;
+				}
+			}
+			if(config.skipFrame === undefined) {
+				docutil.updateStyle(this.skipButton, {'display': 'none'});
+				docutil.updateStyle(this.skipFrame, {'display': 'none'});
+				docutil.updateAttrs(this.skipFrame, {'disabled': 'disabled'});
```

Comment:
Looks like there's an idea to support skipping on a per-game basis, but I don't see why it shouldn't be supported for all games. They are all set up to be deterministic and frame-based.